apps/wallet-mobile/src/features/Send/useCases/ConfirmTx/Summary/ReceiverInfo.tsx

```diff
@@ -26,7 +26,7 @@ export const ReceiverInfo = ({target}: Props) => {
         <>
           <View style={{flexDirection: 'row'}}>
             <Text style={styles.value}>
-              {receiver.selectedNameServer ? nameServerName[receiver.selectedNameServer] : ''}:
+              {receiver.selectedNameServer != null ? nameServerName[receiver.selectedNameServer] : ''}:
```

Comment:
eslint was not complaining I'm a little bit scared, if there are other stuff like this around, this week we should bump rn and eslint/ts